Makefile

```diff
@@ -2,7 +2,7 @@ GPU=1
 OPENCV=1
 DEBUG=0
 
-ARCH= --gpu-architecture=compute_30 --gpu-code=compute_30
+ARCH= --gpu-architecture=compute_20 --gpu-code=compute_20
```

Comment:
No need to change the arch number here as we are not using the original Makefile.